.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [8.0x, 10.x, 12.x]
```

Comment:
The rebase hasn't fully worked? This shouldn't be a part of this PR

Reply:
its fixed now. thanks!